packages/eslint-config-airbnb-base/flat.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * Changes to support the flat config format:
+ * - The `extends` key has been deprecated.
+ * - The `parserOptions` key has been moved under the new `languageOptions` key.
+ * - The `env` key has been deprecated and moved to `languageOptions.globals`.
+ * - The `plugins` key is no longer a list but an object.
+ */
+
+const {
+  extends: airbnbRules,
+  ...airbnbConfig
+} = require('eslint-config-airbnb-base');
+const importPlugin = require('eslint-plugin-import');
+const globals = require('globals');
+
+const mappedEnvs = {
+  es6: 'es2015',
+};
+const handledPlugins = {
```

Comment:
shouldn’t we land a flat entrypoint in the import plugin instead of handling it here?

Reply:
Another argument against implementing it in the plugin side is that in fact the plugin key can be customized, provided that it prefixes the plugin rules. 

So the logic is more for an implementation on the config side more than on the plugin one.

Reply:
Another argument against implementing it on the plugin side is that in fact the plugin key can be customized, provided that it prefixes the plugin rules. 

ESlint flat config allows to do the following:
```
plugin: { importPlugin },
rules: { ˋ${importPlugin}/no-unresolved`: 'off' }
```

So the logic is more for an implementation on the config side more than on the plugin one.

Reply:
what i mean is, eslint-config-airbnb-base should be able to import a single object that can be spread into a flat config, that Just Works.